### DISCUSSION TITLE
a11y Consulting

### DISCUSSION THREAD

**Discussion by zhaohuanyuu** [kobaltedev/kobalte#455] [General]
Why not use solid-aria as the base library for a11y logic? Is it because of the reference to airakit?

**Comment by GiyoMoon** [comment]
Kobalte is basically the successor of solid-aria. It's from the same developers and solid-aria isn't maintained anymore as they moved to Kobalte.

**Reply by zhaohuanyuu** [reply]
Thank you for your reply. I am also looking forward to the best practices of a11y in kobtle.
